src/pages/filter/ui/query-builder/modal-text-editor.tsx

```diff
@@ -161,6 +161,7 @@ export const ModalTextEditor = observer(
           theme={theme}
           isTextEditor
           handleChangeTheme={handleChangeTheme}
+          data-testid={'modal-header'}
```

Comment:
Looks like you use selector string here and in your tests. I'd say it should be only 1 place with it.

Reply:
I've  just overlooked it, fixed.